### PR TITLE
AT_02.05_002 | Redirection to wiki occurs after clicking the 'Learn more about distributed builds' link

### PR DESCRIPTION
**Pull request by mahveenya**
https://trello.com/c/K7VVAWYd/102-at0205002-redirection-to-wiki-occurs-after-clicking-the-learn-more-about-distributed-builds-link

### CODE REVIEW

**Comment by mahveenya** [comment]
The test itself is in the first commit, forgot that we can't change e2e.js and forgot to make the second commit distinct from the 1st one.

File: cypress/e2e/learnMore.cy.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+describe("AT_02.05_002 | Homepage Link 'Learn more about distributed builds'", () => {
```
Comment (Rom-T) [inline cypress/e2e/learnMore.cy.js:1]: Please rename describe and file to Homepage.

Comment (mahveenya) [reply cypress/e2e/learnMore.cy.js]: done

**Review by Rom-T — APPROVED** [review]

**Review by IrynaKolh — DISMISSED** [review]

**Review by puh50 — APPROVED** [review]

**Review by puh50 — COMMENTED** [review]
I'm sorry but why you have created 2 files with different names with the same code inside?
I mean there should be 1 file for one US with only "describe" inside with tests in it.
Perhaps you need to remove one of the file?
I have the screen when reviewed it
image

**Comment by mahveenya** [comment]
@puh50 Hi, i'm not sure which two files you're talking about, there was "homepaje.cy.js" in main, i had the same file with my test in it, then i pulled main to my branch, resolved conflict by adding my test from my "homepaje.cy.js" file to the "homepaje.cy.js" from main

**Comment by puh50** [comment]
> @puh50 Hi, i'm not sure which two files you're talking about (could you provide their names?), there was "homepaje.cy.js" in main, i had the same file with my test in it, then i pulled main to my branch, resolved conflict by adding my test from my "homepaje.cy.js" file to the "homepaje.cy.js" from main

The two files are in the screen I left above )

**Comment by puh50** [comment]
> @puh50 Hmm.. how come i can't see two of them? here are screenshots from the last commit and from the e2e folder of the branch: Screenshot 2023-05-11 at 15 39 41 Screenshot 2023-05-11 at 15 40 32

Really, you are right.
Don't know what it was. Kind of magic)

**Review by puh50 — APPROVED** [review]

**Review by SawkaQA — APPROVED** [review]

**Review by Rom-T — APPROVED** [review]

**Review by SawkaQA — APPROVED** [review]

**Comment by mahveenya** [comment]
@Penshin thanks, man)

**Pull request merged**